queries.sql

```diff
@@ -8,3 +8,43 @@ SELECT name,escape_attempts FROM animals WHERE weight_kg > 10.5 ;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 ;
+
+
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+SELECT * FROM animals;
+UPDATE animals SET species = 'pokemon' WHERE name NOT LIKE '%mon';
+SELECT * FROM animals;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+SAVEPOINT SAVEPOINT1;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+ROLLBACK TO SAVEPOINT1;
+SAVEPOINT SAVEPOINT2;
+UPDATE animals SET weight_kg=weight_kg* -1;
+ROLLBACK TO SAVEPOINT2;
+UPDATE animals SET weight_kg=weight_kg* -1;
```

Comment:
- [ ] This query on line 41 should also include `WHERE weight_kg < 0;` as the query is to find the weight less than 0 too.